apis/config/v1beta1/configuration_types.go

```diff
@@ -169,6 +169,18 @@ type ControllerMetrics struct {
 	// metrics will be reported.
 	// +optional
 	EnableClusterQueueResources bool `json:"enableClusterQueueResources,omitempty"`
+
+	// Configuration for exposing LocalQueue Metrics
+	// +optional
+	LocalQueueMetrics *LocalQueueMetrics `json:"localQueueMetrics,omitempty"`
+}
+
+type LocalQueueMetrics struct {
+	// Optional field which ensures LocalQueue metrics are only gathered for local queues
+	// which match the defined selector. An empty selector matches all LocalQueues.
+	// Defaults to nil which matches no LocalQueues.
+	// +optional
+	LocalQueueSelector *metav1.LabelSelector `json:"localQueueSelector,omitempty"`
```

Comment:
nit; Selector? Since it is already nested in structure with "LocalQueue" name

Reply:
The proposed spec also has a namespace selector so this is to disambiguate